Video.js

```diff
@@ -481,6 +481,10 @@ Video.propTypes = {
   progressUpdateInterval: PropTypes.number,
   useTextureView: PropTypes.bool,
   hideShutterView: PropTypes.bool,
+  cache: PropTypes.shape({
+    dir: PropTypes.string,
```

Comment:
Yes that's a nice one. I set up some defaults in this commit: https://github.com/react-native-video/react-native-video/commit/fe426b2840e715eaff9560a82dbf3e1680e00977:
- When no cache dir is given, use subdirectory "videos" inside app cache directory provided by context. 
- When no cache max size is given, use a default of 100 MB.

Reply:
I made some changes so that the cache directory can be changed on the fly in JS. And also to allow fully disabling of the cache. The code relies on @ReactProp being invoked but if you remove the prop in JS, that method is never invoked. In order to allow disabling (when being enabled), `cache={null}` should be provided. If there's a better way please let me know.